specs/protocol/derivation.md

```diff
@@ -424,6 +445,10 @@ where:
 [RLP format]: https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/
 [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
 
+The Delta upgrade introduced an additional batch type, [span batches][span-batches].
+
+[span-batches]: span-batches.md
```

Comment:
Similarly to how span batches are referenced here, we should also reference the Fjord Brotli spec from the main derivation spec file.